rpcserver.go

```diff
@@ -2664,6 +2664,7 @@ func createRPCOpenChannel(r *rpcServer, graph *channeldb.ChannelGraph,
 		ChanStatusFlags:       dbChannel.ChanStatus().String(),
 		LocalChanReserveSat:   int64(dbChannel.LocalChanCfg.ChanReserve),
 		RemoteChanReserveSat:  int64(dbChannel.RemoteChanCfg.ChanReserve),
+		Tweakless:             dbChannel.ChanType.IsTweakless(),
```

Comment:
should be exposed as "safu" on rpc?

Reply:
heh, maybe? 

Or to be more descriptive, perhaps `CommitTweakless` here?

Reply:
PR description has rationale 
> IMO "safu commitments" is gets the point across a bit better, 

so would assume that would the same for information expressed on the RPC

Reply:
Why not stick to the spec and call it `static_remote_key`?

Reply:
I guess next in-line will be "super safu" channels? ;) 

Moved to be static key, but just because it's clear outside the context of this PR. In a few other areas we deviate from the spec to use subjectively more descriptive names (see the chan reest messages in `lnwire` as an example). We can still use "safu" colloquially when talking about or describing the feature.